pkg/cmd/bootstrap.go

```diff
@@ -259,7 +259,7 @@ func setAccessToken(io *BootstrapParameters) error {
 	if io.GitHostAccessToken != "" {
 		err := ui.ValidateAccessToken(io.GitHostAccessToken, io.ServiceRepoURL)
 		if err != nil {
-			return fmt.Errorf("Please enter a valid access token for --save-token-keyring: %v", err)
+			return fmt.Errorf("Access token validation failed for --save-token-keyring: %v", err)
```

Comment:
We see this error message even when `--save-token-keyring` is not used. Should we get rid of it in the message?
```suggestion
			return fmt.Errorf("Access token validation failed: %v", err)
```

Reply:
Sounds good to me. Done.